src/nvim/message.c

```diff
@@ -237,17 +237,15 @@ msg_strtrunc (
   return buf;
 }
 
-/*
- * Truncate a string "s" to "buf" with cell width "room".
- * "s" and "buf" may be equal.
- */
-void trunc_string(char_u *s, char_u *buf, int room, int buflen)
+/// Truncate a string "s" to "buf" with cell width "room".
+/// "s" and "buf" may be equal.
+void trunc_string(char_u *s, char_u *buf, size_t room, size_t buflen)
 {
-  int half;
-  int len;
-  int e;
-  int i;
-  int n;
+  size_t half;
+  size_t len;
+  size_t i;
+  size_t e;
+  size_t n;
 
   room -= 3;
```

Comment:
It seems unlikely, the only place I'm not certain it can't is in line 234 of `message.c`. Although I may be wrong. I could place an assert just in case. What do you think?